variables.tf

```diff
@@ -379,6 +418,27 @@ variable "os_disk_type" {
   nullable    = false
 }
 
+variable "outbound_ip_address_ids" {
+  type        = list(any)
+  description = "(Optional) The ID of the Public IP Addresses which should be used for outbound communication for the cluster load balancer."
+  default     = null
+  nullable    = true
+}
+
+variable "outbound_ip_prefix_ids" {
+  type        = list(any)
+  description = "(Optional) The ID of the outbound Public IP Address Prefixes which should be used for the cluster load balancer."
+  default     = null
+  nullable    = true
+}
+
+variable "outbound_ports_allocated" {
+  type        = number
+  description = "(Optional) Number of desired SNAT port for each VM in the clusters load balancer. Must be between 0 and 64000 inclusive. Defaults to 0"
```

Comment:
We also need to backquote the number as the [official document](https://github.com/hashicorp/terraform-provider-azurerm/blob/f8e47e77eaa8baf00918d6172edefe97fdfaaa7b/website/docs/r/kubernetes_cluster.html.markdown?plain=1#L583) did here.